runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -238,13 +243,17 @@ sub prepareData {
 		return 0;
 	};
 
+	#Setting outputted workloadNum to empty string if only one workload exists
+	my $workloadCount = $self->workloadDriver->{workloadCount};
+	my $outputWorkloadNum = $workloadCount > 1 ? "Workload $workloadNum," : "workload";
+
 	$logger->debug("prepareData users = $users, logPath = $logPath");
 	print $logHandle "prepareData users = $users, logPath = $logPath\n";
 
 	if (!$self->startDataManagerContainer($users, $logHandle)) {
 		$console_logger->info(
 				    "Could not start AuctionDataManager pod for appInstance "
-				  . "$appInstanceNum of workload $workloadNum." );
+				  . "$appInstanceNum of $outputWorkloadNum" );
```

Comment:
Hi--yeah the removal of the word 'workload' was done on purposes and embedded within the variable. The reason why not all use the same variable ($outputWorkloadNum) is because some outputs only required to conditionally show the number, rather than the number + the word 'workload.'